lib/pf/Switch.pm

```diff
@@ -3112,12 +3112,22 @@ Takes into account the active/active clustering and centralized deauth
 =cut
 
 sub deauth_source_ip {
-    my ($self) = @_;
+    my ($self,$dst_ip) = @_;
+    my $chi = pf::CHI->new(namespace => 'route_int');
+    my $int = $chi->compute($dst_ip, sub {
```

Comment:
We need to validate that the result of this is a valid interface configured in PacketFence